src/common/components/Badge/Badge.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import CloseIcon from "@/images/ic_close.svg";
+import styles from "./badge.module.scss";
+import classNames from "classnames";
+
+interface BadgeProps extends React.HTMLAttributes<HTMLDivElement> {
+  color?: "blue" | "green" | "red";
+  mobile?: boolean;
+  close?: boolean;
+  onClick?: React.MouseEventHandler<HTMLDivElement>;
+  className?: string; // 사용자가 추가할 수 있는 클래스 이름
```

Comment:
꼭 필요하지 않은 주석은 사용하지 않는게 좋습니다!
```suggestion
  className?: string;
```